docs/zkapps/tutorials/anonymous-message-board.mdx

```diff
@@ -2,187 +2,198 @@
 
-You'll need the following installed to use the zkApp CLI:
+## Prerequisites
 
-- NodeJS 16+ (or 14 using `--experimental-wasm-threads`)
-- NPM 6+
-- Git 2+
+Ensure your environment meets the [Prerequisites](/zkapps/tutorials#prerequisites) for zkApp Developer Tutorials.
 
-If you have an older version installed, we suggest installing a newer version
-using the package manager for your system: [Homebrew](https://brew.sh/) (Mac),
-[Chocolatey](https://chocolatey.org/) (Windows), or apt/yum/etc (Linux). On
-Linux, you may need to install a recent NodeJS version via NodeSource
-([deb](https://github.com/nodesource/distributions#debinstall) or
-[rpm](https://github.com/nodesource/distributions#rpminstall)), as recommended
-by the NodeJS Project.
-
-#### Installation
+In particular, make sure you have the zkApp CLI installed:
 
 ```sh
 $ npm install -g zkapp-cli
 ```
 
-#### Usage
-
-```sh
-$ zk --help
-```
-
-##### Create a new project
-
-```sh
-$ zk project message-board  # or path/to/message-board
-
-✔ Fetch project template
-✔ Initialize Git repo
-✔ NPM install
-✔ Set project name
-✔ Git init commit
-
-Success!
+This tutorial has been tested with:
 
-Next steps:
-  cd message-board
-  git remote add origin <your-repo-url>
-  git push -u origin main
-```
+- [zkApp CLI](https://github.com/o1-labs/zkapp-cli) version 0.17.2
+- [o1js](https://www.npmjs.com/package/o1js) version 0.16.2
 
-This command creates a directory containing a new project template, fully set up
-& ready for local development.
+## Create the message-board project
 
-- See the included [README](https://github.com/o1-labs/zkapp-cli/tree/main/templates/project-ts#readme) for usage instructions.
-  All usual commands will be available: `npm run build`, `npm run test`,
-  `npm run coverage`, etc.
-- A Git repo will be initialized in the project directory automatically. For
-  consistency, we use `main` as the default Git branch, by convention.
-- A [Github Actions CI workflow](https://github.com/o1-labs/zkapp-cli/tree/main/templates/project-ts/.github/workflows/ci.yml) is
-  also included. If you push your project to Github, Github Actions will run
-  your tests (named as `*.test.js`) automatically, whenever you push a commit or
-  open a pull request.
-- Code style consistency (via Prettier) and linting (via ES Lint) are
-  automatically enforced using Git pre-commit hooks. This requires no
-  configuration and occurs automatically when you commit to Git--e.g. `git commit -m 'feat: add awesome feature'`.
-- To skip all checks in the Git pre-commit hook (not recommended), you can pass
-  the `-n` flag to Git--e.g. `git commit -m 'a bad commit' -n`. But we'd
-  recommend avoiding this and resolving any errors which exist in your project
-  until the pre-commit hook passes.
+1. Create or change to a directory where you have write privileges.
+1. Create the project:
 
-### Scaffolding
+  ```sh
+  $ zk project zk file message
```

Comment:
```suggestion
  $ zk project
```

Reply:
We are entering `zk file message` after generating the project on line 86 here: 

https://github.com/o1-labs/docs2/pull/672/files#diff-5c1e099bcebf366480c8bc9a70999357c91a82eb788f248aaa766cfe55262a66R86